presto-accumulo/src/test/java/com/facebook/presto/accumulo/TestAccumuloDistributedQueries.java

```diff
@@ -0,0 +1,503 @@
+//                        "SELECT UUID() AS uuid, DATE '2000-01-01', 1234567890, 1.23",
+//                "SELECT count(*) + 1 FROM orders");
+    }
+
+    @Override
+    public void testDelete()
+            throws Exception
+    {
+        try {
+            // TODO Deletes are not supported by the connector
+            super.testDelete();
+        }
+        catch (Exception e) {
+            assertUpdate("DROP TABLE test_delete");
+            assertEquals("This connector does not support updates or deletes", e.getMessage());
+        }
+    }
+
+    @Override
+    public void testInsert()
+            throws Exception
+    {
+        @Language("SQL") String query = "SELECT UUID() AS uuid, orderdate, orderkey FROM orders";
+
+        assertUpdate("CREATE TABLE test_insert AS " + query + " WITH NO DATA", 0);
+        assertQuery("SELECT count(*) FROM test_insert", "SELECT 0");
+
+        assertUpdate("INSERT INTO test_insert " + query, "SELECT count(*) FROM orders");
+
+        assertQuery("SELECT orderdate, orderkey FROM test_insert", "SELECT orderdate, orderkey FROM orders");
+        // Override because base class error: Cannot insert null row ID
+        assertUpdate("INSERT INTO test_insert (uuid, orderkey) VALUES ('000000', -1)", 1);
+        assertUpdate("INSERT INTO test_insert (uuid, orderdate) VALUES ('000001', DATE '2001-01-01')", 1);
+        assertUpdate("INSERT INTO test_insert (uuid, orderkey, orderdate) VALUES ('000002', -2, DATE '2001-01-02')", 1);
+        assertUpdate("INSERT INTO test_insert (uuid, orderdate, orderkey) VALUES ('000003', DATE '2001-01-03', -3)", 1);
+
+        assertQuery("SELECT orderdate, orderkey FROM test_insert",
+                "SELECT orderdate, orderkey FROM orders"
+                        + " UNION ALL SELECT null, -1"
+                        + " UNION ALL SELECT DATE '2001-01-01', null"
+                        + " UNION ALL SELECT DATE '2001-01-02', -2"
+                        + " UNION ALL SELECT DATE '2001-01-03', -3");
+
+        // UNION query produces columns in the opposite order
+        // of how they are declared in the table schema
+        assertUpdate(
+                "INSERT INTO test_insert (uuid, orderkey, orderdate) " +
+                        "SELECT UUID() AS uuid, orderkey, orderdate FROM orders " +
+                        "UNION ALL " +
+                        "SELECT UUID() AS uuid, orderkey, orderdate FROM orders",
+                "SELECT 2 * count(*) FROM orders");
+
+        assertUpdate("DROP TABLE test_insert");
+    }
+
+    @Override
+    public void testRenameColumn()
+            throws Exception
+    {
+        // Override because base class error: Must have at least one non-row ID column
+        // Casting to BIGINT -- mvn test: integers... intellij: longs?  weird
+        assertUpdate("CREATE TABLE test_rename_column AS SELECT CAST(123 AS BIGINT) x, 456 a", 1);
```

Comment:
The object returned from `getOnlyElement(materializedRows.getMaterializedRows()).getField(0)` was an `Integer` (as expected) when run with Maven but was a `Long` value when run in IntelliJ.  Odd behavior that I never fully investigated.

This no longer seems to be the case so I can just use the integer type.  Still needs to be overridden, though, because the connector requires at least two columns to be defined.